extensions/kagi-search/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "kagi-search",
+  "title": "Kagi Search",
+  "description": "Searches the Closed Beta of Kagi Search, using the autosuggestions url.",
+  "icon": "command-icon.png",
+  "author": "fearoffish",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Kagi Search",
```

Comment:
```suggestion
      "title": "Search Kagi",
```

Reply:
The name of the search engine is Kagi Search, it's what they show in the HTML `<title>` too. Do you still suggest I change it?

Reply:
Also, the Google Search extension has the same naming convention (I used that as my template). But, if you'd prefer I change it then I'm happy to.

Reply:
Done

Reply:
> Done

Ah yeah, my bad, I had something else in my head here! Kagi Search is the right thing!

Apologies! 🙏

Reply:
Actually, I thought about it...and I think *you're* right! I want to "Search Kagi" but the title of the extension is "Kagi Search". haha, let's go with the change :)

Reply:
my brain when I want to use the extension says "let's **S**earch **K**agi" not the other way round.

Reply:
I think I confused myself again, hahaha, oh well 😂😂😂